utils/lists.ts

```diff
@@ -462,4 +462,9 @@ export const derivativesList: Record<string, string>[] = [
     description: "Unique 8bit songs for every Loot holder",
     url: "https://www.loottunes.fun/",
   },
+  {
+    name: "Loot Land",
+    description: "Loot Land are parts of the world in Loot metaverse",
+    url: "https://opensea.io/collection/landproject"
```

Comment:
Nit: Can you end a comma in the end to match the styling. Otherwise LGTM!

Reply:
done.